db/migrate/20150828182207_generate_tokens_for_existing_users.rb

```diff
@@ -0,0 +1,7 @@
+class GenerateTokensForExistingUsers < ActiveRecord::Migration
+  def change
+    User.all.each do |user|
```

Comment:
Considering I'll be changing the way the tokens are created (only when the reset password link is generated), I can delete this altogether.